appeals/e2e/cypress/support/extensionMethods.js

```diff
@@ -0,0 +1,8 @@
+// @ts-nocheck
+
+String.prototype.exactMatch = function () {
+	return new RegExp(`^\\s*${this}\\s*$`, 'm');
+};
```

Comment:
In general we'd avoid editing any prototypes, can this be implemented with a util function instead?